filters/color-overlay/src/ColorOverlayFilter.ts

```diff
@@ -27,15 +27,19 @@ type Color = number | number[] | Float32Array;
 class ColorOverlayFilter extends Filter
 {
     private _color = 0x0;
+    private _alpha = 0;
 
     /**
      * @param {number|Array<number>} [color=0x000000] - The resulting color, as a 3 component RGB e.g. [1.0, 0.5, 1.0]
+     * @param {number} [alpha=0] - The alpha value of the color
      */
-    constructor(color: Color = 0x000000)
+    constructor(color: Color = 0x000000, alpha: number)
     {
         super(vertex, fragment);
         this.uniforms.color = new Float32Array(3);
+        this.uniforms.alpha = alpha;
```

Comment:
You don't need to set this here, The uniforms is set by setting `alpha` below.

Reply:
doh, you're right. removed